.gitignore

```diff
@@ -14,4 +14,5 @@ node_modules/
 /npm-debug.log
 
 .parcel-cache
+.cache
```

Comment:
We never want cached stuff in `git`. These files are only important on our local machines.